util/test-chain-utils/src/median_time.rs

```diff
@@ -5,7 +5,10 @@ use ckb_types::{
     prelude::*,
 };
 
-/// TODO(doc): @chuijiaolianying
+/// There's a consensus rule to verify that the block timestamp must be larger than
+/// the median timestamp of the previous 37 blocks.
```

Comment:
The number of latest blocks to calculate the "median time" is specified via `BlockMedianTimeContext::median_block_count()`. [MockMedianTime says `11`](https://github.com/nervosnetwork/ckb/pull/2469/files#diff-44a2f5373da905a8c7c3fa4ccfdbe233f2e8db21799c417f2889e2460dd7cc22R17-R19)